### ISSUE TITLE
get_boxscore doesn't seem to be working, may be related to Issue #91

### ISSUE THREAD

**Issue by professor042** [lbenz730/ncaahoopR#92]
Sorry to bother again, but I think there's an issue with get_boxscore() for all games I've tested.  This issue started in the last day or two.  I wish I knew more about R and could help to troubleshoot.

At first thought the issue applied to games on/after 1/11/23 only but it looks like games before that have the same error as well.

I get this error for each game I've tried:

```
> ncaahoopR::get_boxscore(401470295)
Error in if (is.character(txt) && length(txt) == 1 && nchar(txt, type = "bytes") <  : 
  missing value where TRUE/FALSE needed
```

Another example for a game on 1/10/23 which I previously ran successfully:

```
> ncaahoopR::get_boxscore(401482813)
Error in if (is.character(txt) && length(txt) == 1 && nchar(txt, type = "bytes") <  : 
  missing value where TRUE/FALSE needed
```

![image](https://user-images.githubusercontent.com/20288335/212476857-b10dfedc-7b69-4297-8685-d4c6b4e86b7e.png)

**Comment by lbenz730** [comment]
I think if you update the package this was fixed in #91

**Comment by professor042** [comment]
confirmed, thanks again
